apps/fluent-tester/src/FluentTester/TestComponents/MenuButton/MenuButtonTest.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import * as React from 'react';
+import { Text, View, Switch } from 'react-native';
+import { Separator, MenuButton, ContextualMenuItemProps, ContextualMenuProps } from '@fluentui/react-native';
+import { MENU_BUTTON_TESTPAGE } from './consts';
+import { Test, TestSection, PlatformStatus } from '../Test';
+
+const menuButton: React.FunctionComponent<{}> = () => {
+  const [lastMenuItemClicked, setLastMenuItemClicked] = React.useState(null);
+
+  const [focusOnMount, setShouldFocusOnMount] = React.useState(true);
+  const toggleFocusOnMount = React.useCallback((value) => setShouldFocusOnMount(value), [setShouldFocusOnMount]);
+
+  const [focusOnContainer, setShouldFocusOnContainer] = React.useState(false);
+  const toggleFocusOnContainer = React.useCallback((value) => setShouldFocusOnContainer(value), [setShouldFocusOnContainer]);
+
+  const onItemClick = React.useCallback(
+    (key) => {
+      setLastMenuItemClicked(key);
+    },
+    [setLastMenuItemClicked],
+  );
+  const testImage = require('../Button/icon_24x24.png');
+
+  const menuItems: ContextualMenuItemProps[] = [
+    {
+      itemKey: '1',
+      text: 'MenuItem 1',
+      icon: testImage,
+    },
+    {
+      itemKey: '2',
+      text: 'MenuItem 2',
+    },
+    {
+      itemKey: '3',
+      text: 'MenuItem 3',
+      disabled: true,
+    },
+  ];
+
+  const contextualMenuProps: ContextualMenuProps = {
+    accessibilityLabel: 'MenuButton',
```

Comment:
Would this not break accessibility since it's an English string being used for every language we support?

Reply:
We use English strings for `accessibilityLabel` elsewhere in the Fluent Tester app so I would assume this is fine. Should our controls be supporting multiple languages in the Fluent Tester? I wasn't aware of that. If it's something you think we should fix, maybe we should open an issue to address it.

Reply:
It shouldn't be here. It's good that you noticed it. I used this line for testing, wIll remove it

Reply:
Ah, I just noticed this is only for our test. That's fine I suppose, sorry!

Reply:
I also, didn't notice that it was in Test. I was afraid that I left it in MenuButton.tsx